_docs/_latest/use/model.md

```diff
@@ -20,11 +20,11 @@ $ forseti model create --inventory_index_id <INVENTORY_INDEX_ID> <MODEL_NAME>
 
 The command above will create a data model `<MODEL_NAME>` from inventory `<INVENTORY_INDEX_ID>`.
 
-Note: The status of the Data model can be `SUCCESS`, `PARTIAL_SUCCESS` and `BROKEN`, most likely
-you will be getting `PARTIAL_SUCCESS` for your data model because there are expected warnings that you
+Note: The status of the data model can be `SUCCESS`, `PARTIAL_SUCCESS` and `BROKEN`.
+It is likely you will be getting `PARTIAL_SUCCESS` for your data model because there are expected warnings that you
 might be getting. For example, roles in their alpha version will not contain any permissions and we
 will log that as a warning in the data model. You can see all the errors/warnings of the data model
-with the `get` command. Model with status = `PARTIAL_SUCCESS` is completely safe to use.
+with the `get` command. Model with status = `PARTIAL_SUCCESS` is safe to use.
```

Comment:
To display all the errors and warnings of the data model, use the `get` command."

Reply:
done